ansible/playbooks/koala/activestorage-migration.yml

```diff
@@ -0,0 +1,95 @@
+---
+- hosts: "all"
+  remote_user: "ansible"
+  force_handlers: true
+  become: true
+  become_user: "root"
+
+  vars_prompt:
+    - name: "confirmation"
+      default: "n"
+      prompt: "Are you sure? 'Yes' to continue."
+      private: false
+
+  tasks:
+    - assert:
+        that:
+          - "confirmation == 'Yes'"
+
+    - name: "install new dependency"
+      apt:
+        pkg: "mupdf-tools"
+        state: "present"
+
+    - name: "enable koala's maintenance mode"
+      include_tasks: ".maintenance-on.yml"
+
+    - name: "make pre-upgrade database backup to S3"
+      systemd:
+        name: "backup-databases.service"
+        state: "started"
+
+    - name: "make pre-upgrade file backup to S3"
+      systemd:
+        name: "backup-websites.service"
+        state: "started"
+
+    - name: "update Koala code"
```

Comment:
Shouldn't we run this as `koala` as well, to not mess up file perms?

Reply:
Good one, changing